src/Evolution/DgSubcell/SetInterpolators.hpp

```diff
@@ -285,14 +285,16 @@ struct SetInterpolators {
           const Mesh<Dim> new_mesh{new_extents, new_basis, new_quads};
```

Comment:
Could you split this into 2 commits. One that changes the interpolant and the other that adds it to the input file? Please feel free to squash into final form, including any changes you make from my suggestions.